modules/light-clients/08-wasm/testing/values.go

```diff
@@ -10,7 +10,7 @@ import (
 	clienttypes "github.com/cosmos/ibc-go/v8/modules/core/02-client/types"
 	commitmenttypes "github.com/cosmos/ibc-go/v8/modules/core/23-commitment/types"
 	ibctm "github.com/cosmos/ibc-go/v8/modules/light-clients/07-tendermint"
-	testing "github.com/cosmos/ibc-go/v8/testing"
+	ibctesting "github.com/cosmos/ibc-go/v8/testing"
```

Comment:
This is peculiar. Leaving `testing` as alias would make the linter complain about redundant alias. But removing will make the references to `testing` fail.

Reply:
Thanks for the explanation!